identity/handler.go

```diff
@@ -211,6 +217,7 @@ func (h *Handler) list(w http.ResponseWriter, r *http.Request, _ httprouter.Para
 			IdsFilter:                    r.URL.Query()["ids"],
 			CredentialsIdentifier:        r.URL.Query().Get("credentials_identifier"),
 			CredentialsIdentifierSimilar: r.URL.Query().Get("preview_credentials_identifier_similar"),
+			OrganizationID:               x.ParseUUID(r.URL.Query().Get("organization_id")),
```

Comment:
Here, an invalid UUID will silently be treated as if the parameter was not set at all. This might be hard to debug.

I'd prefer a `400 Bad Request` to be returned if the `organization_id` parameter was set, but the value could not be parsed into an UUID. WDYT?

Reply:
Sure, SGTM.

We don't do any of this validation in the `ids` parameter, either. In fact, we don't even parse the IDs as UUIDs, and instead pass them straight to the query. That means, in practice, that an invalid UUID will return a 500 Internal Server Error. Fix for that is here: https://github.com/ory/kratos/pull/4116

Reply:
Nice catch, approved :)